helm/charts/hydra/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
-  - name: hydra-maester
-    version: 0.0.1
+  - name: ory.sh/hydra-maester
```

Comment:
Is this intended? The chart name is `hydra-maester` right?

Reply:
Yes, since it's no longer directly included in the `charts/` directory, it needs the repo name added so helm can pull it down when installing

Reply:
Not sure, this appears to be causing:

```
Error: found in requirements.yaml, but missing in charts/ directory: ory.sh/hydra-maester
```